hxrsnd/attocube.py

```diff
@@ -275,7 +275,7 @@ def reset(self, ret_status=False, print_set=True):
         return self._status_print(status, "Reset motor '{0}'".format(
             self.desc), ret_status=ret_status, print_set=print_set)
     
-    def move(self, position, check_status=True, *args, **kwargs):
+    def move(self, position, check_status=True, timeout=0, *args, **kwargs):
```

Comment:
I started using the `timeout` attribute for general `set` calls and have it set to 1 second. This unfortunately means it only gives motions 1 second to complete as well. So I just have it set to 0 because I don't have a good way to tell how long a motion should take. I initially started using the velocity and distance to travel, but it wasn't reliable across motors when you factor in backlash, acceleration, etc. This was a long way of saying, setting timeout to 0 got things working but shouldn't be a final solution.